lineman/app/components/group_page/group_page_controller.coffee

```diff
@@ -8,9 +8,15 @@ angular.module('loomioApp').controller 'GroupPageController', ($rootScope, $rout
     $rootScope.$broadcast 'setTitle', @group.fullName()
     $rootScope.$broadcast 'analyticsSetGroup', @group
     MessageChannelService.subscribeTo("/group-#{@group.key}")
+    @setPhotos()
   , (error) ->
     $rootScope.$broadcast('pageError', error)
 
+  @setPhotos = =>
```

Comment:
This is the only thing not currently working; the `coverStyle` is being set, but not updated in the template. Apparantly I don't quite get how ng-style works or something.